components/org.wso2.carbon.identity.idle.account.identification/src/main/java/org/wso2/carbon/identity/idle/account/identification/constants/IdleAccIdentificationConstants.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.idle.account.identification.constants;
+
+/**
+ * Constants for idle account identification.
+ */
+public class IdleAccIdentificationConstants {
+
+    public static final String IDLE_ACC_IDENTIFICATION_SERVICE_ERROR_PREFIX = "IDLE_ACC-";
+
+    public static final String LAST_LOGIN_TIME_CLAIM = "http://wso2.org/claims/identity/lastLogonTime";
+    public static final String EMAIL_CLAIM = "http://wso2.org/claims/emailaddress";
+    public static final String DATE_INACTIVE_AFTER = "inactiveAfter";
+    public static final String DATE_EXCLUDE_BEFORE = "excludeBefore";
+    public static final String DATE_FORMAT_REGEX = "^\\d{4}-\\d{2}-\\d{2}$";
+    public static final String DEFAULT_USER_STORE_NAME = "DEFAULT";
+
+    /**
+     * Class containing SQL queries.
+     */
+    public static class SQLConstants {
+
+        public static final String GET_INACTIVE_USERS_FROM_SPECIFIC_DATE =
+                "SELECT USER_NAME, DATA_VALUE FROM IDN_IDENTITY_USER_DATA WHERE " +
+                        "DATA_KEY = ? AND TENANT_ID = ? AND DATA_VALUE < ?";
+
+        public static final String GET_LIMITED_INACTIVE_USERS_FROM_SPECIFIC_DATE =
+                "SELECT USER_NAME, DATA_VALUE FROM IDN_IDENTITY_USER_DATA WHERE " +
+                        "DATA_KEY = ? AND TENANT_ID = ? AND DATA_VALUE < ? AND DATA_VALUE > ?";
```

Comment:
does the < and > operators work for date values in all the databases

Reply:
The last login time will be stored as an epoch string. Also, most relational databases support these operators. Since, ours identity DB is a mssql DB, there won't be an issue.